### PR TITLE
Customer service: Telegram bot

### PR DESCRIPTION
**Pull request by DanieloM83**
<h1>Customer service telegram bot</h1>

**Telegram bot and HTTP endpoint were developed for the following purposes:**

1. **Wait for an HTTP POST request** with information about the order;
2. When receiving a request, the **bot sends all received data to the Telegram channel** in one message in the format:
    "✅ **{Теперішня дата і час} Нове замовлення на {К-сть ящиків}.**
    🧑 **ПІБ**: {ПІБ}
    📧 **Електронна адреса**: {Email}
    📞 **Номер телефону**: {Номер телефону}
    🏠 **Адреса**: {Адреса}
    💬 **Коментар**: {Коментар}"

### CODE REVIEW

File: customer_service/telegram_bot/bot/bot.py
Comment (flop-code) [inline customer_service/telegram_bot/bot/bot.py:1]: Remove `TEST_API_KEY`

Comment (flop-code) [inline customer_service/telegram_bot/bot/bot.py:1]: Remove `TEST_API_KEY`

File: customer_service/telegram_bot/bot/config.py
Comment (flop-code) [inline customer_service/telegram_bot/bot/config.py:1]: Remove `TEST_API_KEY`

File: customer_service/telegram_bot/routes/routes.py
Comment (flop-code) [inline customer_service/telegram_bot/routes/routes.py:1]: No need to return something from the router.

**Review by flop-code — CHANGES_REQUESTED** [review]
А few minor details need to be worked out.

File: customer_service/telegram_bot/routes/routes.py
Comment (DanieloM83) [reply customer_service/telegram_bot/routes/routes.py:1]: This is for testing, since there is no error handler, let it be at least for some information

**Review by flop-code — APPROVED** [review]
Well done.

**Pull request merged**